### DISCUSSION TITLE
modifying model class and training

### DISCUSSION THREAD

**Discussion by ramkirank** [NVIDIA-NeMo/NeMo#2899] [Q&A]
I want to modify model architecture of matchbox1D for command detection (like adding more linear or rnn) and train. How can i do this if configuration file don't provide these options?

**Comment by VahidooX** [comment]
Changing the architecture of a model usually needs modification of the corresponding class (EncDecClassificationModel here) and can not be done by the configs. You may duplicate this model and make your own changes.
